rllib/utils/metrics/__init__.py

```diff
@@ -34,6 +34,7 @@
 COMPUTE_GRADS_TIMER = "compute_grads"
 GARBAGE_COLLECTION_TIMER = "garbage_collection"
 SYNCH_WORKER_WEIGHTS_TIMER = "synch_weights"
+SYNCH_ENV_CONNECTOR_STATES_TIMER = "synch_env_connectors"
```

Comment:
Should we warn users in case connector synchs take longer than MAX_S seconds?

Reply:
I think it's fine. We don't warn them either if e.g. remote worker weight synching takes more than n sec.